ReactAndroid/src/main/java/com/facebook/react/touch/JSResponderHandler.java

```diff
@@ -70,7 +72,7 @@ public boolean onInterceptTouchEvent(ViewGroup v, MotionEvent event) {
       // Therefore since "UP" event is the last event in a gesture, we should just let it reach the
       // original target that is a child view of {@param v}.
       // http://developer.android.com/reference/android/view/ViewGroup.html#onInterceptTouchEvent(android.view.MotionEvent)
-      return v.getId() == currentJSResponder;
+      return reactTagFor(v) == currentJSResponder;
```

Comment:
I'm thinking `getReactTag` might actually be more obvious. Somehow when I see `reactTagFor` I'm assuming it's going to do a lookup in a map. On the other hand "get" implies a simple (cheap) getter.